compiler/rustc_session/src/options.rs

```diff
@@ -708,6 +709,16 @@ macro_rules! options {
                 None => false,
             }
         }
+
+        fn parse_wasi_exec_model(slot: &mut Option<WasiExecModel>, v: Option<&str>) -> bool {
+            match v {
+                Some("command")  => *slot = Some(WasiExecModel::Command),
+                Some("reactor") => *slot = Some(WasiExecModel::Reactor),
+                Some(_) => return false,
+                None => {}
```

Comment:
Hmm, this case is `-Z wasi-exec-model` without an argument, it should probably also `return false`.

Reply:
Oh, I guess I thought that was no wasi-exec-model arg passed at all.